frontend/modelarchive/src/test/resources/models/logging/service.py

```diff
@@ -45,8 +48,10 @@ def inference(model_input):
         :param model_input: transformed model input data
         :return: inference results
         """
-        logging.info("LoggingService inference [PID]: %d", os.getpid())
-        return ["{} OK".format(os.getpid())] * len(model_input)
+        time.sleep(0.01)
+        LoggingService.counter += 1
+        logging.info("LoggingService inference [PID]: %d seq=%d ", os.getpid(), LoggingService.counter)
```

Comment:
Is this counter required? Was this added by accident?

Reply:
The counter is for testing